### PR TITLE
(doc): Enabling flowcharts

### PR DESCRIPTION
**Pull request by tamil777selvan**
## Proposed changes

Fix - #6540

- Enabling flowcharts

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -65,6 +65,11 @@ module.exports = {
                 label: 'API',
                 position: 'left',
                 docId: 'api',
+            }, {
+                type: 'doc',
+                label: 'Flowchart',
+                position: 'left',
+                docId: 'flowchartsoverview',
```
Comment (christian-bromann) [inline website/docusaurus.config.js:72]: I don't think we want to have this part of the main menu. Let's drop a section into the [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) and link to the page.

**Review by christian-bromann — COMMENTED** [review]
These flowcharts probably need some updates but that can be part of another PR. One comment, otherwise looks good to me

File: website/docusaurus.config.js (hunk 1 above)
Comment (tamil777selvan) [reply website/docusaurus.config.js:72]: @christian-bromann, Should we position the flowchart above the "Proposing a Change" section, or do you think it should be relocated under a different section? Please share your thoughts.

Screenshot 2023-09-07 at 9 42 06 PM

Comment (christian-bromann) [reply website/docusaurus.config.js:72]: This looks good to me 👍

**Review by christian-bromann — APPROVED** [review]
Awesome, LGTM 👍

**Pull request merged**